classes/DirtEventTypes.sc

```diff
@@ -91,7 +92,7 @@ DirtEventTypes {
 						midiout.control(chan, 99, nrpnMSB);
 						midiout.control(chan, 98, nrpnLSB);
 						midiout.control(chan, 6,  valMSB);
-						midiout.control(chan, 38, valLSB)
+						midiout.control(chan, 38, valLSB);
```

Comment:
no, I just found it looks better because it makes the series of commands look the same.

If there is a rule about the trailing semicolon, it is that it is good style to definitely leave it out where the return value of the function is used. In cases of multiple side effects it's fine to leave it. But all that isn't very strongly followed, so a bit of a matter of taste.